CSV_Modifier_Client/Controllers/HomeController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Diagnostics;
+using CSV_Modifier_Client.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CSV_Modifier_Client.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly ILogger<HomeController> _logger;
```

Comment:
This is not used anywhere

Reply:
it comes with template, didn't add/remove anything here...